core/src/test/java/io/github/tcdl/msb/monitor/aggregator/HeartbeatTaskTest.java

```diff
@@ -33,17 +35,34 @@
     @Before
     public void setUp() {
         when(mockObjectFactory.createRequester(anyString(), any(RequestOptions.class))).thenReturn(mockRequester);
-        @SuppressWarnings("unchecked")
-        Callback<List<Message>> any = any(Callback.class);
-        when(mockRequester.onEnd(any)).thenReturn(mockRequester);
+        when(mockRequester.onRawResponse(any(Callback.class))).thenReturn(mockRequester);
+        when(mockRequester.onEnd(any(Callback.class))).thenReturn(mockRequester);
     }
 
+    @SuppressWarnings("unchecked")
     @Test
     public void testRun() {
         heartbeatTask.run();
 
+        ArgumentCaptor<Callback> onResponseCaptor = ArgumentCaptor.forClass(Callback.class);
+        ArgumentCaptor<Callback> onEndCaptor = ArgumentCaptor.forClass(Callback.class);
+        ArgumentCaptor<List> messageCaptor = ArgumentCaptor.forClass(List.class);
```

Comment:
nice if to name this smth like receivedMessagesCaptor or at least messagesCaptor